sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+
+Specific SIG deliverables are as per the above, and the [general SIG responsibilities
+set out by the CNCF TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+
+## Current CNCF Projects considered to be within the Scope of this SIG
+
+
+
+1. Kubernetes
+2. Containerd
+3. Harbor
+4. Dragonfly
+5. Virtual Kubelet
+6. CRI-O
+7. KubeEdge
+8. KubeVirt
+
+
+## Interfaces With Other Related Groups
+
+
+
+*   **[Several Kubernetes SIGs](https://github.com/kubernetes/community)**
+    cover Kubernetes-specific workload, scheduling, autoscaling, execution
+    and other related abstractions, interfaces, and implementations of
+    these interfaces.  We will maintain communication with these Kubernetes
+    SIGs where needed.  Our aim is to avoid unnecessary duplication of
+    effort by the two groups and maintain clear and consistent messaging
+    to our end user community and projects.
+*   **[CNCF Security SIG](https://github.com/cncf/sig-security)**
+    - works on the more general area of cloud-native security including a
+    uthentication, authorization, encryption, accounting, auditing, and
+    related topics.  We defer as much as possible to this group to deal
+    with general security-related issues and liaise closely with them on
+    how to deal with security areas where these arise.
+*   **[CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery)**
+    is focussed on the development, deployment, operation and testing of
+    cloud-native applications.  We collaborate with this SIG where it
+    pertains to helping to ensure that the required underlying workload
+    execution abstractions and mechanisms are suitably provided to support
+    these application-level delivery needs.
+*   **Note regarding Container Registries/Repositories** and the like:
+    While image packaging and distribution (and hence container
+    registries/repositories in general) fall within the scope of this
+    Runtime SIG, many of their common features and use cases are better
+    dealt with by other CNCF SIGS.  For example:
+    *   image storage, caching, etc - Storage SIG
+    *   Image encryption, signing etc - Security SIG
+    *   use of image registries to store and distribute many other types
+        of artifacts, and in particular the format of these artifacts,
+	including helm charts, OPA policies, public security certificates,
+	data sets, machine learning models, etc, etc - the SIG relevant to
+	those artifact types, e.g. Apps SIG, Security SIG, etc.
+
+
+## Operating Model
+
+This SIG follows the [standard operating guidelines](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#operating-model)
+provided by the TOC unless otherwise stated here.
+
+**Current TOC Liaison:**  Brendan Burns and Brian Grant
+
+**[Proposed] Co-Chairs:** [Quinton Hoole](https://www.linkedin.com/in/quintonhoole/),
+  [Ricardo Aravena](https://www.linkedin.com/in/raravena), 1 TBD
```

Comment:
I am Diane Feddema and I'm interested in being a SIG-Runtime co-chair.  I'm a Principal Software Engineer at Red Hat in the AI Center of Excellence in the CTO's office.  I have a background in HPC and have been working on AI/ML applications and benchmarking at Red Hat for 7 years.  I'm interested in this role because I'm working with our hardware partners to enable hardware accelerators (GPU, FPGAs, etc) that will run on Kubernetes/Openshift. www.linkedin.com/in/dianefeddema

Reply:
Thanks Diane.  I think that the SIG would greatly appreciate your assistance.  Any objections?

Reply:
@dfeddema to formalize this I've added you as a nominee and up for a vote to be discussed in the next TOC meeting. As per https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections